web/src/index.js

```diff
@@ -2,20 +2,39 @@ import React from 'react';
 import ReactDOM from 'react-dom';
 import './index.css';
 import App from './App.jsx';
-import { store } from './config/store';
+import { initStore } from './config/store';
 import { Provider } from 'react-redux';
 import { BrowserRouter } from "react-router-dom";
 import * as serviceWorker from './serviceWorker';
 
+// we should know better, but just in case we navigate to the unused react dev server, show an alert.
+if(window.location.origin === 'http://localhost:3000') alert("Wrong port. Go to http://localhost:3030 to load the app from the server")
```

Comment:
nice

Reply:
does it still open a new browser window at 3000 when you run start?

Reply:
Nope, that is disabled by adding `devServer.open = false` on craco.config.js